cassandra-migration/src/main/java/org/cognitor/cassandra/migration/Database.java

```diff
@@ -241,4 +297,73 @@ public Database setConsistencyLevel(ConsistencyLevel consistencyLevel) {
         this.consistencyLevel = notNull(consistencyLevel, "consistencyLevel");
         return this;
     }
+
+    /**
+     * Attempts to acquire the lead on a migration through a LightWeight
+     * Transaction.
+     *
+     * @param repositoryLatestVersion
+     *            the latest version number in the migration repository
+     * @return if taking the lead succeeded
+     */
+    boolean takeLeadOnMigrations(int repositoryLatestVersion) {
+        if (VersionNumber.parse("2.0").compareTo(cassandraVersion) > 0) {
```

Comment:
Is it by intention that you say here "> 2.0" while in createSchemaTable you say ">= 2.0" ?
If not maybe you can create a method "isIfNotExistsSupported" or "isBeforeV2"

Reply:
Both are actually valid.
Here we're checking that 2.0 > cassandraVersion (meaning we're using a version before 2.0), and return true directly because LWTs aren't available.
In `createSchemaTable` we're checking that cassandraVersion >= 2.0 to add `IF NOT EXISTS` to the statement.
I agree these conditions are hard to read, and I'll build a single method to test where we are comparing to 2.0, that I'll use in both places.